src/main/java/redis/clients/jedis/args/UnblockType.java

```diff
@@ -0,0 +1,21 @@
+package redis.clients.jedis.args;
+
+import redis.clients.jedis.util.SafeEncoder;
+
+/**
+ * Unblock type for {@code CLIENT UNBLOCK} command.
+ */
+public enum UnblockType implements Rawable {
+  TIMEOUT, ERROR;
+
+  private final byte[] raw;
+
+  UnblockType() {
+    raw = SafeEncoder.encode(this.name().toLowerCase());
```

Comment:
Any reason to make it lower?
```suggestion
    raw = SafeEncoder.encode(this.name());
```

Reply:
No, Let me remove it.